src/js/components/Nav/Nav.tsx

```diff
@@ -163,18 +165,19 @@ const FlyOutNode = ({ item }) => {
  * @param acc an array
  * @param cur a flyout item
  */
-const reducer = (acc: [], cur) => {
-  return [...acc, <FlyOutNode key={cur.title} item={cur} />];
-};
+const reducer = (tree:[], openItems: boolean) => {
+  return tree.reduce((acc: [], cur:any)  => [...acc, <FlyOutNode key={cur.title} item={cur} openItems={openItems} />], []);
+}
 
-const generateFlyoutContents = (flyoutSections: any) => {
+const generateFlyoutContents = (flyoutSections: any, openItems: boolean) => {
   return flyoutSections.map((section, idx) => {
     return (
-      <section className="u-borderTop" key={idx}>
+      <section className="u-borderTop u-spaceBottomSm" key={idx}>
         <div className="Nav-sectionHeading u-colorNeutral7 u-textUppercase u-textBold u-fontSizeXs">
           {section.heading}
         </div>
-        <div>{section.tree.reduce(reducer, [])}</div>
+        {/* <div className="Nav-sectionItems">{section.tree.reduce(buildReducer, [])}</div> */}
```

Comment:
If we dont need this anymore, lets go ahead and delete it :)

Reply:
Remove code